communications/meetings.md

```diff
@@ -0,0 +1,67 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+
+- Determine the scope and minimum length of the meeting.
+- Give all participants as much context as possible to make an informed decision on RSVP.
+- Gather topics and issues for agenda items that are relevant to the scope of the meeting,
+done either individually or through crowdsourcing to increase accuracy and minimize bias
+- Asynchronously resolve as many agenda items as possible before the meeting.
+
+Guide:
+
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+- Create the meeting agenda on Hugo for the meeting, provide sufficient time ahead of the meeting to accommodate all participants.
+    - For a net-new meeting, the organizer(s) should define the initial scope of agenda items.
+    - For a recurring meeting with past sessions, use the agenda/notes items from the previous meeting session as a starting point. Remove ones no longer needed or already resolved.
+- Initiate one or more meeting note(s) on Hugo for meeting agenda discussions.
+    - Send these notes to the relevant Slack channel.
+    - Edit meeting agenda to reflect the outcome of these discussions.
+- In each meeting note, participants write down briefing, information, knowledge, and ideas that are relevant.
```

Comment:
Need to mention about participants commenting on agenda note in Slack about briefing, information, etc.
Suggestion:
`On the slack message for the agenda note, participants comment with briefing, information, knowledge and ideas that are relevant`

Then would recommend having the tip about Hugo doing a good job at syncing notes right after the point 'Sending these notes to the relevant Slack channel' right after this point.

Reply:
the Note feature from Hugo can be sync'ed to Slack (both ways), so I wouldn't limit where participants want to carry out the discussion (ultimately does not matter as much). I guess I can incorporate this point as a subpoint under this one.

Reply:
nvm, there's a point above already covered sending to Slack channel.

perhaps we need to organize this, as it's a bit too wordy.

Reply:
Oh right, yes I agree. then better to say:
On Hugo or on Slack, participants comment on the agenda note with briefing, information, knowledge and ideas that are relevant